app/src/main/java/com/tasomaniac/openwith/settings/SettingsFragment.java

```diff
@@ -226,11 +227,10 @@ private void displayLicensesDialogFragment() {
     }
 
     private void startContactEmailChooser() {
-        ShareCompat.IntentBuilder.from(getActivity())
-                .addEmailTo("Said Tahsin Dane <tasomaniac+openlinkwith@gmail.com>")
-                .setSubject(getString(R.string.app_name))
-                .setType("message/rfc822")
-                .startChooser();
+        EmailIntentBuilder.from(getActivity())
+                .to("tasomaniac+openlinkwith@gmail.com")
```

Comment:
I'm not entirely sure if that's supported with `mailto:` URIs (see [RFC 6068](https://tools.ietf.org/html/rfc6068)).

`mailto:Said%20Tahsin%20Dane%20%3Ctasomaniac%2Bopenlinkwith%40gmail.com%3E` is not valid.
`mailto:?to=Said%20Tahsin%20Dane%20%3Ctasomaniac%2Bopenlinkwith%40gmail.com%3E` might be.

However, I haven't actually tested this and `EmailIntentBuilder` currently doesn't allow you to create such URIs.